cdap-api/src/main/java/co/cask/cdap/api/app/AbstractApplication.java

```diff
@@ -214,4 +216,21 @@ protected void addService(Service service) {
   protected void addService(String name, HttpServiceHandler handler, HttpServiceHandler...handlers) {
     configurer.addService(new BasicService(name, handler, handlers));
   }
+
+  /**
+   * Adds a {@link Schedule} to the Application
+   * @param schedule the schedule to be added
+   */
+  protected void addSchedule(Schedule schedule) {
+    configurer.addSchedule(schedule);
+  }
+
+  /**
+   * Schedule the specified {@link Workflow}
+   * @param schedule the name of the schedule to be added for the Workflow
+   * @param workflow the name of Workflow
+   */
+  protected void scheduleWorkflow(String schedule, String workflow) {
```

Comment:
Any reason why we do not use ``Schedule`` and ``Workflow`` classes for parameter types here?